plugins/python-build/bin/python-build

```diff
@@ -1486,6 +1487,29 @@ use_xcode_sdk_zlib() {
   fi
 }
 
+use_tcltk() {
+   # check args passed, use that otherwise check for homebrew tcl-tk
+   local tcltk_libdir="$(brew --prefix tcl-tk 2>/dev/null || true)"
+   local regex_tcltk_libs_args="(--with-tcltk-libs='([^']+)')"
+
+   if [[ "$PYTHON_CONFIGURE_OPTS" =~ $regex_tcltk_libs_args ]]; then
+     local tcltk_libs_match="${BASH_REMATCH[1]}"
+     local tcltk_match_quoted="${tcltk_libs_match//--with-tcltk-libs=/}"
+     PYTHON_CONFIGURE_OPTS="${PYTHON_CONFIGURE_OPTS//$tcltk_libs_match/}"
```

Comment:
This extracting-and-readding is limiting and unreliable. In particular, it requires exactly quoted single quotes and exactly around the value -- which is something one has no way to know or expect.

Better properly parse the configure values into an array, e.g. with `xargs` as per https://stackoverflow.com/questions/37372225/convert-a-string-into-an-array-with-bash-honoring-quotes-for-grouping.